web/js/app.js

```diff
@@ -0,0 +1,9 @@
+require('jquery');
+require('../../node_modules/material-design-lite/dist/material.min.js');
```

Comment:
Since the `material-design-lite` package.json specifies to import this file, `'material-design-lite' should be enough here.

Reply:
This dependency was not found: `material-design-lite in ./web/js/app.js` maybe with an import this would work. Leaving it as it is now.

Reply:
That is strange, as it does work for jquery.